cirq/contrib/acquaintance/permutation.py

```diff
@@ -112,14 +114,14 @@ class LinearPermutationGate(PermutationGate):
     def __init__(self,
                  num_qubits: int,
                  permutation: Dict[int, int],
-                 swap_gate: ops.Gate=ops.SWAP
-                 ) -> None:
+                 swap_gate: Optional[ops.Gate] = None) -> None:
```

Comment:
I've fixed the bug in the formatter where it would compare against a too-recent version of master. You should be able to revert almosta ll the formatting changes.